__tests__/comments.ts

```diff
@@ -330,7 +330,10 @@ describe('query commentPreview', () => {
     expect(res.errors).toBeFalsy();
     expect(res.data.commentPreview).toMatchSnapshot();
 
-    const mention = '@Lee';
+    await con
+      .getRepository(User)
+      .update({ username: 'Lee' }, { username: 'Lee-' });
+    const mention = '@Lee-';
```

Comment:
Maybe suggest to append something at the back for safety tests

Reply:
Good point. Let me add it as a separate one.